### PR TITLE
Several fixes from review of website

### PR DESCRIPTION
**Pull request by mikewalch**
- Improved Apache branding
- Updated download page to make it clear that older releases of Fluo,
  fluo-dev, and zetten are not part of ASF.
- Added warning to release page that instructions are historical and
  will be updated after release.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -3,26 +3,26 @@ layout: page
 title: Documentation
 ---
 
-[Fluo] and [Fluo Recipes] have seperate documentation as they are different repositories with their own release cycle.
+[Apache Fluo] and [Apache Fluo Recipes] have seperate documentation as they are different repositories with their own release cycle.
 
-## Fluo
+## Apache Fluo
 
 Latest release is `{{ site.latest_fluo_release }}` made on {{ site.latest_fluo_release_date }}.
```
Comment (keith-turner) [inline docs/index.md]: Since the latest release is currently not an Apache release, need to point this out.

Comment (mikewalch) [inline docs/index.md]: Good point.  Also, this should be noted in archives of release notes, release summaries, and API.  I will work on this and push another commit.

File: pages/release-summaries-archive.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ---
 layout: page
-title: Fluo Release Summaries Archive
+title: Release Summaries Archive
 permalink: "/release-summaries/"
 ---
 
-Below are the release summaries for all Fluo releases:
+Below are the release summaries for all Apache Fluo releases:
```
Comment (keith-turner) [inline pages/release-summaries-archive.md]: Currently there are no Apache Fluo Releases

File: pages/release-notes-archive.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 ---
 layout: page
-title: Fluo Release Notes Archive
+title: Release Notes Archive
 permalink: "/release-notes/"
 ---
 
-Below are the release notes for all Fluo releases:
+Below are the release notes for all Apache Fluo releases:
```
Comment (keith-turner) [inline pages/release-notes-archive.md]: Currently there are not Apache Fluo release to have release notes for ... so this is misleading....

Comment (keith-turner) [inline pages/release-notes-archive.md]: In addition to making things clear on the pages that link to info about old releases... we should probably also add something to the pages that actually have the content

File: docs/index.md [hunk 4]
```diff
@@ -3,26 +3,30 @@ layout: page
...
+The first release of Apache Fluo has not been made yet.
 
-* Fluo documentation - [Latest][fluo-docs-latest] \| [Archive][fluo-docs-archive]
-* Fluo API - [Latest][fluo-api-latest] \| [Archive][fluo-api-archive]
-* Fluo release summary -  [Latest][fluo-sum-latest] \| [Archive][fluo-sum-archive]
-* Fluo release notes - [Latest][fluo-notes-latest] \| [Archive][fluo-notes-archive]
+Latest non-ASF release is `{{ site.latest_fluo_release }}` made on {{ site.latest_fluo_release_date }}.
 
-## Fluo Recipes
+* Documentation - [Latest][fluo-docs-latest] \| [Archive][fluo-docs-archive]
```
Comment (keith-turner) [inline docs/index.md]: Would be nice to add a notice about non asf release to the content of the older release docs.

Comment (mikewalch) [inline docs/index.md]: Added in ffbfbff94dd53427

**Comment by keith-turner** [comment]
I created mikewalch/incubator-fluo-website#1 with some more updates.

**Comment by keith-turner** [comment]
+1

**Comment by keith-turner** [comment]
+1 again

**Pull request merged**